test/storage/e2e_test.dart

```diff
@@ -98,7 +101,7 @@ void main() {
         var r2 = await storage.deleteBucket(bucketName);
         expect(r2, isNull);
       }
-    });
+    }, skip: 'unable to test with uniform buckets enforced for account');
```

Comment:
Is there a plan to reenable this test in some way?  By testing if uniform buckets are enforced, and putting a guard?  If there is no issue filed, or plan to enable them, then are these just kept to allow local testing, by removing the skips?

Reply:
I have no plans... if we get hold of a test account that doesn't have uniform buckets enforced re-enabling them seems reasonable. I still have an bug open to resolve test account issues.